CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+## Development Workflow
+
+1. Create an issue or pick an available open issue.
+2. Assign the issue to yourself.
+3. In waffle (https://waffle.io/zrts/eosportal-front) move the issue to "In Progress".
```

Comment:
Not big deal, but I would keep the workflow fully in GH using https://github.com/zrts/eosportal-front/projects/1 (Projects)

Another thing, is that I think only people added to the repo can move items in kanban. Perhaps we need to write some guidelines for external contributors who are not part of the repo. 👍